packages/core/src/components/DynamicGrid/index.tsx

```diff
@@ -0,0 +1,101 @@
+import withStyles, { css, WithStylesProps } from '@airbnb/lunar/lib/composers/withStyles';
+import React from 'react';
+
+type State = {
+  items: number;
+};
+
+export type GridProps = {
+  // List of width/item pairs, describes how many items to display for windows of at least that width.
+  breakpoints?: { [key: number]: number };
+  // Items per row for screens smaller than the smallest breakpoint.
+  defaultItems?: number;
+  // Padding between items in units.
+  padding?: number;
+};
+
+type Props = GridProps & WithStylesProps;
+
+class DynamicGrid extends React.PureComponent<Props, State> {
+  public static defaultProps = {
+    breakpoints: {},
+    defaultItems: 1,
+    padding: 2,
```

Comment:
hrmm maybe no `padding` prop so the padding between columns can be made consistent with `Grid`'s `Col` padding? a `noPadding` prop if no padding is desired

Reply:
You could set `padding` to `0`, does that work?
![Screen Shot 2019-06-11 at 1 47 35 PM](https://user-images.githubusercontent.com/8676510/59305656-8862eb80-8c4f-11e9-9cfb-5b7554f08a7c.png)

Reply:
i'm suggesting no `padding` prop and baking in the same column padding as Grid/Col components. `noPadding` was another suggestion if that's a use case you would need for this. my concern is with keeping grid spacing consistent within the design system.